PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -70,21 +70,34 @@
         'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
     )
 )
-MicroEventContentMC = cms.PSet(
-    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
-)
-MicroEventContentMC.outputCommands += [
-        'keep *_slimmedGenJets*_*_*',
+
+MicroEventContentGEN = cms.PSet(
+    outputCommands = cms.untracked.vstring(
         'keep patPackedGenParticles_packedGenParticles_*_*',
         'keep recoGenParticles_prunedGenParticles_*_*',
         'keep LHEEventProduct_*_*_*',
-        'keep PileupSummaryInfos_slimmedAddPileupInfo_*_*',
         'keep GenFilterInfo_*_*_*',
         'keep GenLumiInfoHeader_generator_*_*',
         'keep GenLumiInfoProduct_*_*_*',
         'keep GenEventInfoProduct_generator_*_*',
+        'keep *_slimmedGenJetsFlavourInfos_*_*',
+        'keep *_slimmedGenJets__*',
+        'keep *_slimmedGenJetsAK8__*',
+        'keep *_genMetTrue_*_*',
         # RUN
```

Comment:
From the changes above:
```diff
+        'keep *_slimmedGenJetsFlavourInfos_*_*',
+        'keep *_slimmedGenJetsAK8__*',
```
do not correspond to existing producer configurations in 80X
I suggest to drop these.

```diff
 +        'keep *_genMetTrue_*_*', 
```
leads to addition of a new product to miniAOD, which is in violation of no-change policy for the production release event content.
May I suggest to drop it here and push it back to MicroEventContentGEN at the end of the file so that the miniAOD content is not modified and the new miniGEN still keeps its content appropriately.